components/ui/Footer.js

```diff
@@ -18,47 +27,67 @@ export const Footer = () => {
       {/* TASK - React 1 week 2 */}
       {/* Create a new List for the Pages */}
       {/* We need to use the <Link /> component here */}
-      {/* <div className={styles.pages}>
+      <div className={styles.pages}>
         <h3>Pages</h3>
-        <ul>
-          <li> <Link/> </li>
-          ...
-        </ul>
-      </div> */}
-      {/* Docs for the Link: https://nextjs.org/docs/pages/api-reference/components/link */}
-
-      {/* TASK - React 1 week 1 */}
-      {/* Add a new list item for LINKEDIN */}
-      <div className={styles.footerLinks}>
-        <h3>Follow us</h3>
-        <ul className={styles.footerList}>
+        <ul className={styles.pagesList}>
           <li>
-            <a href="https://facebook.com">Facebook</a>
+            {" "}
```

Comment:
You don't need these forced empty spaces
```suggestion
            
```

Reply:
I removed all {'' ''}